src/components/Journey/Journey.css

```diff
@@ -1,60 +1,29 @@
 .journey-section{
   background-color: #F0C808;
-  /* height: 85vh; */
-}
-
-.container{
   height: 85vh;
-  display: grid;
-  grid-template-columns: repeat(5, 1fr);
-  grid-template-rows: repeat(5, 1fr);
 }
 
-.information-container{
-  grid-area: 1/ 1/ 3 / 3;
-  display: inline-flex;
+.container{
+  height: 79vh;
+  display: flex;
   justify-content: center;
```

Comment:
What is this file in charge of?

Reply:
This is the CSS file for the styling of that component